rag/svr/task_executor_refactor/dataset_wiki_generator.py

```
@@ -871,6 +899,38 @@ async def persist_wiki_pages(
         )
         return
 
+    if insert_errors:
+        logging.warning(
```

Ruff (0.16.1)

[error] Not enough arguments for `logging` format string

(PLE1206)